app/resources/v1/pacbio/runs/well_pool_resource.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module V1
+  module Pacbio
+    module Runs
+      # WellPoolResource
+      class WellPoolResource < JSONAPI::Resource
+        model_name 'Pacbio::WellPool'
+
+        has_one :pool
+      end
+    end
+  end
+end
```

Comment:
I know we exposed Well:Library before, but I'm not sure there's much value in exposing what is essentially a join-table. We should probably hide this from the API as an implementation decision.